test/sanity/code-smell/use-compat-six.py

```diff
@@ -12,24 +12,24 @@ def main():
         # requirement on six so it needs to try importing six to give the correct error message
         'lib/ansible/modules/cloud/digital_ocean/digital_ocean.py',
         # correct imports in the following files and remove them from this list
-        'contrib/inventory/apache-libcloud.py',
-        'contrib/inventory/cobbler.py',
-        'contrib/inventory/collins.py',
-        'contrib/inventory/consul_io.py',
-        'contrib/inventory/ec2.py',
-        'contrib/inventory/freeipa.py',
-        'contrib/inventory/lxd.py',
-        'contrib/inventory/nova.py',
-        'contrib/inventory/nsot.py',
-        'contrib/inventory/packet_net.py',
-        'contrib/inventory/proxmox.py',
-        'contrib/inventory/rax.py',
-        'contrib/inventory/rudder.py',
-        'contrib/inventory/scaleway.py',
-        'contrib/inventory/spacewalk.py',
-        'contrib/inventory/vmware.py',
-        'contrib/inventory/vmware_inventory.py',
-        'docs/bin/plugin_formatter.py',
+        # 'contrib/inventory/apache-libcloud.py',
+        # 'contrib/inventory/cobbler.py',
+        # 'contrib/inventory/collins.py',
+        # 'contrib/inventory/consul_io.py',
+        # 'contrib/inventory/ec2.py',
+        # 'contrib/inventory/freeipa.py',
+        # 'contrib/inventory/lxd.py',
+        # 'contrib/inventory/nova.py',
+        # 'contrib/inventory/nsot.py',
+        # 'contrib/inventory/packet_net.py',
+        # 'contrib/inventory/proxmox.py',
+        # 'contrib/inventory/rax.py',
+        # 'contrib/inventory/rudder.py',
+        # 'contrib/inventory/scaleway.py',
+        # 'contrib/inventory/spacewalk.py',
+        # 'contrib/inventory/vmware.py',
+        # 'contrib/inventory/vmware_inventory.py',
+        # 'docs/bin/plugin_formatter.py',
```

Comment:
Remove these (and the now unnecessary comment above) instead of commenting them out.

Reply:
Ooops. I meant to do that.